lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -287,17 +346,66 @@ function gatherTapTargets() {
       return;
     }
 
-    targets.push({
-      clientRects: visibleClientRects,
-      snippet: truncate(tapTargetElement.outerHTML, 300),
-      // @ts-ignore - getNodePath put into scope via stringification
-      path: getNodePath(tapTargetElement),
-      // @ts-ignore - getNodeSelector put into scope via stringification
-      selector: getNodeSelector(tapTargetElement),
-      href: /** @type {HTMLAnchorElement} */(tapTargetElement)['href'] || '',
+    const largestRect = getLargestRect(visibleClientRects);
+    const largestRectCenterPoint = getRectCenterPoint(largestRect);
+    // round so we can can assume whole numbers during in-viewport check
+    largestRectCenterPoint.x = Math.round(largestRectCenterPoint.x);
+    largestRectCenterPoint.y = Math.round(largestRectCenterPoint.y);
+
+    if (largestRectCenterPoint.x >= window.innerWidth) {
+      // we don't scroll sideways, so the center of this tap target is always hidden
+      return;
+    }
+
+    enhancedTapTargets.push({
+      tapTargetElement,
+      largestRectCenterPoint,
+      visibleClientRects,
     });
   });
 
+
+  enhancedTapTargets.sort(
+    (a, b) => {
+      return a.largestRectCenterPoint.y - b.largestRectCenterPoint.y;
+    }
+  );
+
+  // Disable point events so that tap targets below them don't get
+  // detected as non-tappable (they are tappable, just not while the viewport
+  // is at the current scroll position)
+  const reenableFixedAndStickyElementPointerEvents = disableFixedAndStickyElementPointerEvents();
```

Comment:
so elementFromPoint uses pointer events to decide what's on top? interesting, so it really is from the mouse-perspective then, not visual. TIL :)